src/common/dom/selector-path.js

```diff
@@ -34,9 +34,7 @@ export const generateSelectorPath = (elem, targetFields = []) => {
   try {
     while (elem?.tagName) {
       const { id, localName } = elem
-      targetFields.forEach(field => {
-        nearestFields[nearestAttrName(field)] ||= elem[field]
-      })
+      targetFields.forEach(field => { nearestFields[nearestAttrName(field)] ||= (elem[field]?.baseVal || elem[field]) })
```

Comment:
Are there are any other non-SVG attributes that could have this property that we need to be aware of?